packages/components/Button/src/Button.tsx

```diff
@@ -34,11 +34,35 @@ export const buttonLookup = (layer: string, state: IPressableState, userProps: B
   );
 };
 
+export const extractMarginAndroid = (style) => {
+  const marginKeys = [
+    'margin',
+    'marginTop',
+    'marginBottom',
+    'marginLeft',
+    'marginRight',
+    'marginStart',
+    'marginEnd',
+    'marginVertical',
+    'marginHorizontal',
+  ];
+  const extractedMargin = {},
+    mask = {};
+  marginKeys.forEach((key) => {
+    if (style && style[key]) {
+      extractedMargin[key] = style[key];
+      mask[key] = 0;
```

Comment:
I feel like undefined would be better instead, since that's more equivalent to "this property isn't set"